From c38f8e8179bd537e882ac43fec990300611f49d6 Mon Sep 17 00:00:00 2001
From: Yuan Yao <yao.yuan@nxp.com>
Date: Fri, 4 Mar 2016 11:31:56 +0800
Subject: [PATCH 1176/1383] DTS: QDMA: add status-sizes and queue-sizes
 properties in dtsi qdma node

Signed-off-by: Yuan Yao <yao.yuan@nxp.com>
[Original patch taken from QorIQ-SDK-V2.0-20160527-yocto]
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 arch/arm/boot/dts/ls1021a.dtsi                 | 2 ++
 arch/arm64/boot/dts/freescale/fsl-ls1043a.dtsi | 2 ++
 2 files changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/ls1021a.dtsi b/arch/arm/boot/dts/ls1021a.dtsi
index fdd1fe3..64b692d 100644
--- a/arch/arm/boot/dts/ls1021a.dtsi
+++ b/arch/arm/boot/dts/ls1021a.dtsi
@@ -681,6 +681,8 @@
 			interrupt-names = "qdma-error", "qdma-queue";
 			channels = <8>;
 			queues = <2>;
+			status-sizes = <64>;
+			queue-sizes = <64 64>;
 			big-endian;
 		};
 
diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1043a.dtsi b/arch/arm64/boot/dts/freescale/fsl-ls1043a.dtsi
index 0c41018..b4f94b1 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1043a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1043a.dtsi
@@ -972,6 +972,8 @@
 			interrupt-names = "qdma-error", "qdma-queue";
 			channels = <8>;
 			queues = <2>;
+			status-sizes = <64>;
+			queue-sizes = <64 64>;
 			big-endian;
 		};
 
-- 
2.8.1

